src/Analysis/GWASUIMockup/GWASUI.jsx

```diff
@@ -0,0 +1,483 @@
+import { getAllSources, getCohortJsonByName } from '../../cohortMiddleware';
+
+const { Step } = Steps;
+
+const steps = [
+  {
+    title: 'Step 1',
+    description: 'Select a cohort for GWAS',
+  },
+  {
+    title: 'Step 2',
+    description: 'Select harmonized variables for phenotypes and covariates',
+  },
+  {
+    title: 'Step 3',
+    description: 'Select which variable is your phenotype and remove unwanted covariates',
+  },
+  {
+    title: 'Step 4',
+    description: 'Set workflow parameters',
+  },
+  {
+    title: 'Step 5',
+    description: 'Submit GWAS job',
+  },
+];
+
+const GWASUI = (props) => {
+  const [form] = Form.useForm();
+  const [current, setCurrent] = useState(0);
+  const [selectedCohort, setSelectedCohort] = useState(undefined);
+  const [selectedConcepts, setSelectedConcepts] = useState([]);
+  const [numOfPC, setNumOfPC] = useState(3);
+  const [selectedPhenotype, setSelectedPhenotype] = useState(undefined);
+  const [selectedCovariates, setSelectedCovariates] = useState([]);
+  const [jobName, setJobName] = useState(undefined);
+  const [showJobSubmissionResult, setShowJobSubmissionResult] = useState(false);
+  const [jobSubmittedRunID, setJobSubmittedRunID] = useState(undefined);
+  const [marinerJobStatus, setMarinerJobStatus] = useState([
+    {
+      runID: 'run-10000',
+      status: 'completed',
+      output: [
+        {
+          name: 'test_null_model_report.html',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_null_model_report.html',
+        },
+        {
+          name: 'test_null_model_invnorm_report.html',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_null_model_invnorm_report.html',
+        },
+        {
+          name: 'test_single_assoc_qq.png',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_single_assoc_qq.png',
+        },
+        {
+          name: 'test_single_assoc_manhattan.png',
+          url: 'https://qa-mickey.planx-pla.net/dashboard/Secure/hapmap-genesis-public-outputs/test_single_assoc_manhattan.png',
+        },
+      ],
+    },
+  ]);
+  const [sources, setSources] = useState([]);
+  const [cohortData, setCohortData] = useState([]);
+
+
+  const onStep3FormSubmit = useCallback((values) => {
+    setNumOfPC(values.numOfPC);
+  }, []);
+
+  const handleDelete = (key) => {
+    const newlySelectedPhenotype = (selectedPhenotype.concept_id === key) ? undefined : selectedPhenotype;
+    const newlySelectedCovariates = selectedCovariates.filter((item) => item.concept_id !== key);
+    setSelectedConcepts(selectedConcepts.filter((item) => item.concept_id !== key));
+    setSelectedPhenotype(newlySelectedPhenotype);
+    setSelectedCovariates(newlySelectedCovariates);
+    form.setFieldsValue({
+      covariates: newlySelectedCovariates.map((val) => val.name),
+      // outcome: newlySelectedPhenotype.name,
```

Comment:
remove lines of commented out codes, or leave a comment in the code to justify why should we keep it

Reply:
removed commented out code